packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -369,7 +369,11 @@ export class AppManagementClient implements DeveloperPlatformClient {
       directory?: string
     },
   ): Promise<OrganizationApp> {
-    const variables = createAppVars(name, options?.isLaunchable, options?.scopesArray)
+    // query for latest api version
+    const apiVersions = await this.apiVersions(org.id)
+    const apiVersion = apiVersions.publicApiVersions[apiVersions.publicApiVersions.length - 2]
```

Comment:
I'm sure it is always the case, but I wouldn't rely on `unstable` always being last. Can we filter out unstable, sort, and choose the latest? Thats what we do on dev dash